decompiler/config.cpp

```diff
@@ -52,6 +52,14 @@ Config read_config_file(const fs::path& path_to_config_file, const std::string&
   config.streamed_audio_file_names =
       inputs_json.at("streamed_audio_file_names").get<std::vector<std::string>>();
 
+  if (cfg.contains("art_group_dump_file")) {
```

Comment:
thanks for adding this. I don't remember why I didn't do this in the first place.